UI/Shared/PlayerCard.razor

```diff
@@ -0,0 +1,34 @@
+<div class="bg-indigo-900 w-28 h-48 grid content-between justify-items-center rounded-3xl p-2 @GetActiveStateCssClasses(Active)">
+        <div class="rounded-full h-20 w-20">
+            <img src="@Image" />
+        </div>
+        <div class="text font-semibold w-8 h-5">
+            <h1 class="text-white">@Name</h1>
+        </div>
+        <div class="max-w-full max-h-full h-10 w-8 text-center flex items-center justify-center">
+            <img src="@Mark" class="w-full h-full"/>
+        </div>
+</div>
+@if (Active) {
+
+    <div class="text-white text-lg font-semibold">Your turn</div>
+}
+
+@code {
+
+    [Parameter]
+    public string Image { get; set; } = null!;
+
+    [Parameter]
+    public string Name { get; set; } = null!;
+
+    [Parameter]
+    public string Mark { get; set; } = null!;
+
+    [Parameter]
+    public bool Active { get; set; } = false;
+
+    private string GetActiveStateCssClasses(bool Active) {
+        return Active == true ? "border-4 border-white rounded-3xl" : "";
```

Comment:
I believe the border is a bit thick, you can try border-2 and make it less rounded maybe rounded-xl